src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,87 @@
+.navbar {
+  position: fixed;
+  top: 0;
+  background-color: #e1e5f2;
+  width: 100%;
+  transition: 0.5s;
+  z-index: 1000;
+}
+
+.DesktopNavigation {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 15px 80px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+nav ul li {
+  list-style-type: none;
+  display: inline-block;
+}
+
+nav ul li a {
+  padding: 0px 35px;
+  display: block;
+  font-weight: normal;
+}
+
+ul li a:hover {
+  font-weight: 500;
+  color: #257a8b;
+  border-right: 3px solid #257a8b;
+  border-left: 3px solid #257a8b;
+}
+
+.MobileNavigation {
+  display: none;
+}
+
+@media (max-width: 768px) {
+  .DesktopNavigation {
+    display: none;
+  }
+
+  .MobileNavigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 12px 40px 4px 40px;
+    background-color: #dbecff;
+  }
+
+  .navlinks ul {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    position: absolute;
+    border-top: 2px solid #575757;
+    top: 110px;
+    right: 0;
+    height: 150px;
+    width: 100%;
+    background: whitesmoke;
+  }
+
+  .nav-whitespace {
+    padding-bottom: 20px;
+  }
+}
+
+.HamburgerMenu {
+  position: absolute;
+  right: 40px;
+  cursor: pointer;
+}
```

Comment:
Make sure to use consistent naming convention for your CSS classes and IDs. It's usually recommended to use `kebab-case`.
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/